lib/logstash/inputs/kafka.rb

```diff
@@ -175,6 +175,8 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   config :ssl_keystore_password, :validate => :password
   # The password of the private key in the key store file.
   config :ssl_key_password, :validate => :password
+  # Algorithm to use when verifying host. Set to "" to disable
+  config :ssl_endpoint_identification_algorithm, :validate => :string
```

Comment:
Can we set the default value to `https` here, to ensure the default won't change in the (probably somewhat unlikely) event that the Kafka client changes their default value again?